frame/max-encoded-len/derive/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "max-encoded-len-derive"
+version = "3.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+description = "Derive support for MaxEncodedLen"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[lib]
+proc-macro = true
+
+[dependencies]
+frame-support-procedural-tools = { version = "3.0.0", path = "../../support/procedural/tools" }
+proc-macro2 = "1.0.6"
+quote = "1.0.3"
+syn = { version = "1.0.58", features = ["full"] }
+
+[features]
+default = ["std"]
+std = []
```

Comment:
I think we can remove the std feature here, as we don't use it AFAIK.

Reply:
We don't use it in this crate, but adding it here is how we can pass `std`-ness along to `codec` if desired.

Reply:
codec-derive doesn't have any std feature, what does this feature means ?